src/rtp/rtp.c

```diff
@@ -26,19 +27,19 @@
 struct rtp_sock {
 	/** Encode data */
 	struct {
-		uint16_t seq;   /**< Sequence number       */
-		uint32_t ssrc;  /**< Synchronizing source  */
+		uint16_t seq;                /**< Sequence number            */
+		uint32_t ssrc;               /**< Synchronizing source       */
 	} enc;
-	int proto;              /**< Transport Protocol    */
-	void *sock_rtp;         /**< RTP Socket            */
-	void *sock_rtcp;        /**< RTCP Socket           */
-	struct sa local;        /**< Local RTP Address     */
-	struct sa rtcp_peer;    /**< RTCP address of Peer  */
-	rtp_recv_h *recvh;      /**< RTP Receive handler   */
-	rtcp_recv_h *rtcph;     /**< RTCP Receive handler  */
-	void *arg;              /**< Handler argument      */
-	struct rtcp_sess *rtcp; /**< RTCP Session          */
-	bool rtcp_mux;          /**< RTP/RTCP multiplexing */
+	int proto;                           /**< Transport Protocol         */
+	void *sock_rtp;                      /**< RTP Socket                 */
+	void *sock_rtcp;                     /**< RTCP Socket                */
+	struct sa local;                     /**< Local RTP Address          */
+	struct sa rtcp_peer;                 /**< RTCP address of Peer       */
+	rtp_recv_h *recvh;                   /**< RTP Receive handler        */
+	rtcp_recv_h *rtcph;                  /**< RTCP Receive handler       */
+	void *arg;                           /**< Handler argument           */
+	struct rtcp_sess *rtcp;              /**< RTCP Session               */
```

Comment:
the whitespace changes are not strictly needed

Reply:
Not strictly for all the lines. It was only to align them to
```c
       RE_ATOMIC bool rtcp_mux;             /**< RTP/RTCP multiplexing      */
```
Which didn't fit.